### DISCUSSION TITLE
[Plugin Request] scoop plug

### DISCUSSION THREAD

**Discussion by photkey** [Flow-Launcher/Flow.Launcher#1072] [Ideas]
scoop search and install software
Search through the `scoop-search` command (https://github.com/shilangyu/scoop-search), or search for the json file in the directory User\buckets\apps\bucket.
Press Enter on the found results to install the software directly

Manage installed software (update, uninstall, etc.)
View the list of installed software through the `scoop list` command, and then manage it.

**Comment by Garulf** [comment]
I've been meaning to look at scoop as a plugin, I've just haven't gotten around to it yet.

**Reply by photkey** [reply]
Looking forward to your work.
